### PR TITLE
Python Node Editor Visual Update - part #3

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

A continuation of [Part 1](https://github.com/DynamoDS/Dynamo/pull/13732) and [Part 2](https://github.com/DynamoDS/Dynamo/pull/13763). This PR only deals with the reserved words colors, aligning the color scheme used in design script editor and the python editor.

#### Color scheme as per [Figma](https://www.figma.com/file/1q7EWQGYO7pPDhyLf8nuwW/Python-node-editor-restyling?node-id=37%3A62973&t=Jix1pw06z2ChdpYH-0) project
image

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

- aligns design script and python script reserved keywords
- hyperlink highlight color 

### Reviewers

@sm6srw 
@Amoursol 

### FYIs

@Jingyi-Wen

### CODE REVIEW

**Comment by sm6srw** [comment]
Tests passed here: https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/8620/

**Comment by dnenov** [comment]
I think these are the final color changes for the moment and we should be good to Merge if everything else looks OK @sm6srw !

File: src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs [hunk 1]
```diff
@@ -418,4 +419,4 @@ internal void Dispose()
             this.dynamoViewModel.PreferencesViewModel.PropertyChanged -= OnPreferencesViewModelPropertyChanged;
         }
     }
-}
```
Comment (reddyashish) [inline src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs:421]: Looks like no changes to this file.

**Review by reddyashish — COMMENTED** [review]
LGTM once the conflicts are resolved.

File: src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs (hunk 1 above)
Comment (dnenov) [reply src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs:421]: Removed unnecessary namespace statements

**Comment by reddyashish** [comment]
Jobs on master-5 are hanged, running self-service https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/1129/.

**Comment by QilongTang** [comment]
[DynamoCoreWpfTests.CoreUserInterfaceTests.WorkspaceContextMenu_TestIfOpenOnRightClick](https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/8701/testReport/junit/DynamoCoreWpfTests/CoreUserInterfaceTests/WorkspaceContextMenu_TestIfOpenOnRightClick/) sporadit test failure reported. Merging.

**Comment by reddyashish** [comment]
@dnenov  was about to add a fix in this for the folding behavior you mentioned. May be in another PR.

**Comment by QilongTang** [comment]
> @dnenov was about to add a fix in this for the folding behavior you mentioned. May be in another PR.

Great! Let's use another one. I would love to test this before recharge sprint

**Pull request merged**